doc/source/api/bg-mpl-stylesheets.rst

```diff
@@ -0,0 +1,44 @@
+:tocdepth: -1
+
+bg-mpl-stylesheets package
```

Comment:
please check.  Should these be underscores here? snake_case?

Reply:
noted, will double check this tmr